test/dummy/config/application.rb

```diff
@@ -22,4 +22,3 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
   end
 end
-
```

Comment:
👏

(I'll make sure @moonglum will be appropriately punished.)

Reply:
I'm so sorry, sir.